src/scheduler/data_types.h

```diff
@@ -339,6 +345,7 @@ struct server_info
 	 */
 	status *policy;
 	fairshare_head *fairshare;	/* root of fairshare tree */
+	resresv_set **equiv_classes;
```

Comment:
I understand your concern, but don't know what to do about it.  Equivalence classes require all those parameters to function properly.  Can we combine the two set idea into one structure?  Should I rename resresv_set into job_equiv_class?